### PR TITLE
Allow embed of just the clock and Match The clock and the schedule

### PR DESCRIPTION
**Pull request by chegejames**
Additions to allow embed of just the clock with a page /embed.html.

Changes to g.timechart.js to ensure the time clock and schedule below it match in both color and order.

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-taxclock.codeforkenya.org
```
Comment (DavidLemayian) [inline CNAME]: @chegejames We might want to put this back.

File: _config.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 title: TaxClock Kenya
 description: It’s easy to see how much of your salary is taken for taxes, but do you know how many hours are allocated to those deductions? Enter your salary to see how your tax money is being spent.
 
-url: https://taxclock.codeforkenya.org
-enforce_ssl: taxclock.codeforkenya.org
+url: https://chegejames.github.io/TaxClock.Kenya/
+enforce_ssl: chegejames.github.io/TaxClock.Kenya
```
Comment (DavidLemayian) [inline _config.yml]: @chegejames And this back too.

File: js/taxclock/tc-clock.js [hunk 3]
```diff
@@ -59,8 +58,10 @@ jQuery(document).ready(function($) {
   $("#info").click(function(){
     getData("agency", year, salary);
   });
-
+ 
+  
   TC.clock.update = function () {
+  	console.log("update clock");
```
Comment (DavidLemayian) [inline js/taxclock/tc-clock.js]: I don't think needed anymore.

File: test.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+	<body>
+		<div id="embed"></div>
+	</body>
+
+	<footer>
+		
+	</footer>
+	<script src="js/vendor/pym.js"></script>
+		<script>
+			var pymParent = new pym.Parent("embed", 'https://chegejames.github.io/TaxClock.Kenya/embed.html', {});
```
Comment (DavidLemayian) [inline test.html]: Test looks good. Probably point to a relative URL instead?

File: CNAME (hunk 1 above)
Comment (chegejames) [inline CNAME]: @DavidLemayian i've put it back.

File: js/taxclock/tc-clock.js (hunk 3 above)
Comment (chegejames) [inline js/taxclock/tc-clock.js]: @DavidLemayian taken out

File: test.html (hunk 4 above)
Comment (chegejames) [inline test.html]: @DavidLemayian Fixed that too.

File: .project [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+	<name>TaxClock.Kenya</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+	</buildSpec>
+	<natures>
+		<nature>com.aptana.projects.webnature</nature>
+	</natures>
+</projectDescription>
```
Comment (DavidLemayian) [inline .project]: This file probably should be ignored in `.gitignore`

**Pull request merged**